launchpad-frontend/src/jobsPage/jobsPage.js

```diff
@@ -393,9 +393,9 @@ function JobExpanded({ postingSelected, updateList, postings, saved }) {
                     alignContent: "space-evenly",
                   }}
                 >
-                  <Button variant="contained" sx={PageStyles.apply}>
+                  <ApplyButton companyName={expandedPosting.companyName}>
```

Comment:
im getting an error cuz of this. I think you have to import ApplyButton into this file

Reply:
done!! thank u for noticing 🥳